### PR TITLE
{bio}[GCCcore/7.3.0] Add dcm2niix v1.0.20180622

### PR DESCRIPTION
**Pull request by crubb**
This PR adds dcm2niix v1.0.20180622 and it's dependencies, CharLS and OpenJPEG:

> dcm2niix is a designed to convert neuroimaging data from the DICOM format to the NIfTI format.

> CharLS is a C++ implementation of the JPEG-LS standard for lossless and near-lossless image compression and decompression. JPEG-LS is a low-complexity image compression standard that matches JPEG 2000 compression ratios

> OpenJPEG is an open-source JPEG 2000 codec written in C language. It has been developed in order to promote the use of JPEG 2000, a still-image compression standard from the Joint Photographic Experts Group (JPEG). Since may 2015, it is officially recognized by ISO/IEC and ITU-T as a JPEG 2000 Reference Software.

### CODE REVIEW

File: easybuild/easyconfigs/c/CharLS/CharLS-2.0.0-GCCcore-7.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/team-charls/charls/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['528c6a3cc168a44e73f2890d8f4a35104a54d752eba3d6a643f050b72dd67cfa']
+
+builddependencies = [
+    # use same binutils version that was used when building GCC toolchain
+    ('binutils', '2.30', '', True),
```
Comment (boegel) [inline easybuild/easyconfigs/c/CharLS/CharLS-2.0.0-GCCcore-7.3.0.eb]: @crubb No need to use dummy binutils here, please use ('binutils', '2.30') instead

File: easybuild/easyconfigs/d/dcm2niix/dcm2niix-1.0.20180622-GCCcore-7.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/rordenlab/dcm2niix/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['e9f79509f44aac82c9663381f8f4bfb18a9a3c3eb112d418c92629a871bbb13c']
+
+builddependencies = [
+    # use same binutils version that was used when building GCC toolchain
+    ('binutils', '2.30', '', True),
```
Comment (boegel) [inline easybuild/easyconfigs/d/dcm2niix/dcm2niix-1.0.20180622-GCCcore-7.3.0.eb]: @crubb No need to use dummy binutils here, please use ('binutils', '2.30') instead

File: easybuild/easyconfigs/o/OpenJPEG/OpenJPEG-2.3.0-GCCcore-7.3.0.eb [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/uclouvain/openjpeg/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['3dc787c1bb6023ba846c2a0d9b1f6e179f1cd255172bde9eb75b01f1e6c7d71a']
+
+builddependencies = [
+    # use same binutils version that was used when building GCC toolchain
+    ('binutils', '2.30', '', True),
```
Comment (boegel) [inline easybuild/easyconfigs/o/OpenJPEG/OpenJPEG-2.3.0-GCCcore-7.3.0.eb]: @crubb No need to use dummy binutils here, please use ('binutils', '2.30') instead

**Comment by crubb** [comment]
Done!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in this PR)
node2456.golett.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/7c3cbb5dbae1051bfd34efa97915869a for a full test report.

**Comment by boegel** [comment]
Going in, thanks @crubb!

**Pull request merged**